src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-event.ts

```diff
@@ -35,7 +35,7 @@ export function projectEventTest(socket: SocketIO, projData: projectsController.
             "projectID": projData.projectID,
             "timestamp": Date.now(),
             "chunk": chunk,
-            "chunk_total": chunk + Math.random() * 100,
+            "chunk_total": 1,
             "eventArray": []
```

Comment:
If your `eventArray` is an empty array. It would not triggers the update. Since no file is considered `changed`, no build would be triggerred.

Reply:
The old value would triggers the build because Turbine thinks there are more chunks/changedFiles, but the chunk is lost due to network issue. 
So it would trigger build after the timer's timeout reaches.

Reply:
`eventArray` is not empty. We update it dynamically https://github.com/eclipse/codewind/blob/ad1e09bdb0bc2210d5e5ba952f1a35a16a382675/src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-event.ts#L103 and updates do happen